### PR TITLE
Add restore_ts for apply

### PR DESCRIPTION
**Pull request by 3pointer**
1. move file related field to `KVMeta`.
2. add `restore_ts` for `KVMeta` to implement PiTR.

### CODE REVIEW

**Review by joccau — APPROVED** [review]
LGTM

File: proto/import_sstpb.proto [hunk 1]
```diff
@@ -284,10 +284,30 @@ message DuplicateDetectResponse {
...
+
+    // the key ts space large than restore_ts will filter.
+    // because key is sorted by ts already.
+    uint64 restore_ts = 5;
+}
+
+
+message ApplyRequest {
+    // The meta of the KV file.
+    KVMeta meta = 1;
```
Comment (kennytm) [inline proto/import_sstpb.proto:309]: 🤔 use a different number?

Comment (3pointer) [reply proto/import_sstpb.proto:309]: 🤔 maybe it's okay since only `br-stream` use `ApplyReqeust` for now.

Comment (kennytm) [reply proto/import_sstpb.proto:309]: 🤔 okay

**Pull request merged**